api/src/test/java/org/openmrs/module/appointmentscheduling/validator/AppointmentTypeValidatorTest.java

```diff
@@ -25,7 +25,7 @@
 	@Before

```

Comment:
Same comment here in the previous test class... the fact that we have to change all these tests suggests that we are making backwards incompatible changes... we should keep the old tests as-in, and use the new constructor in any new tests where we are explicitly testing the new behavior.